apps/pm-user/src/components/atoms/logo/logo.module.scss

```diff
@@ -0,0 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Italianno&display=swap');
```

Comment:
This is my mistake. I will remove this import statement, as I've already added Italliano and Lato globally.

Reply:
Issue resolved.